### PR TITLE
feat: [FFM-5220]: GO one click code example for onboarding flow

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

A new one-click code example for Golang, when the user chooses the SDK in the onboarding flow.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -706,48 +706,97 @@ onboarding:
     golang: |
       # Supported GO versions
 
-      This version of FFM has been tested with GO 1.14
+      <a href="https://go.dev/doc/install">Golang 1.6</a> or newer (go version)
```
Comment (stephenmcconkey) [inline src/modules/75-cf/strings/strings.en.yaml:709]: Small one, this should be 1.16

**Review by stephenmcconkey — APPROVED** [review]
Looks good to me, just that one very small change needed with the Golang version - Approving with that change in mind

File: src/modules/75-cf/strings/strings.en.yaml (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:709]: Really great spot, thanks

**Review by stephenmcconkey — DISMISSED** [review]

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -711,48 +711,97 @@ onboarding:
...
+      # Test your application
 
       ```
-      showFeature, err := client.BoolVariation(featureFlagKey, &target, false)
+      export FF_API_KEY=<your key here>
+      go run getting_started.go
       ```
 
-      Flush any changes and close the SDK
+      If you dont have the right version of golang installed locally, or don't want to install the dependencies you can use docker to quick get started
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:799]:
```suggestion
      If you don't have the right version of golang installed locally, or don't want to install the dependencies you can use docker to quickly get started
```

File: src/modules/75-cf/strings/strings.en.yaml [hunk 3]
```diff
@@ -711,48 +711,97 @@ onboarding:
...
+        sdkKey   string = getEnvOrDefault("FF_API_KEY", "changeme")
+      )
+
+      func main() {
+        log.Println("Harness SDK Getting Started")
+
+        // Create a feature flag client instance
+        client, err := harness.NewCfClient(sdkKey)
+        if err != nil {
+          log.Fatalf("could not connect to CF servers %s\n", err)
```
Comment (crystal-fong) [inline src/modules/75-cf/strings/strings.en.yaml:759]:
```suggestion
          log.Fatalf("could not connect to FF servers %s\n", err)
```

Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:759]: Not sure about this as I took it from our readme

Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: have changed anyway, can't change the code part though

File: src/modules/75-cf/strings/strings.en.yaml (hunk 2 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml]: thanks have changed :)

**Review by crystal-fong — APPROVED** [review]

**Review by stephenmcconkey — APPROVED** [review]

**Pull request merged**